### ISSUE TITLE
Deprecate `Workplane.findFace`?

### ISSUE THREAD

**Issue by marcus7070** [CadQuery/cadquery#665]
I don't think anyone is going to miss it. Shall we mark it deprecated and remove next release?

**Comment by jmwright** [comment]
@marcus7070 What's wrong with `findFace`? Is it redundant or something?

**Comment by marcus7070** [comment]
Pretty much redundant. It isn't used internally, or in cq-kit or paramak. It didn't have any tests written for it. I just assumed is was an artefact of an old `Workplane.workplane` implementation.

I just went and chased the git blame for it, and I actually see it was added not too long ago by @adam-urbanczyk in 8606e6bb1f4ebdcc2723e1e6059146b18f05d2c6 to support `Workplane.cutThruAll`, and then removed by him in 3348c18889b37b7489468b4a0f59d73494a67389, so I'll leave it up to him if it should be deprecated or not.

To be clear, the only reason I think it should go is to trim 12 lines of code out of the codebase. It doesn't cause any problems or anything.
